### DISCUSSION TITLE
Prefixed tables

### DISCUSSION THREAD

**Discussion by dominiquevienne** [bennett-treptow/laravel-migration-generator#49] [Q&A] [answered]
Hi, 

First of all, thx, this tool is great!

My tables are prefixed (`DB_PREFIX` in .env). Is there a way to get the script get aware of it and not rendering prefixed strings?

Thx

**Comment by bennett-treptow** [comment, marked as answer]
@dominiquevienne I am sorry that I didn't ever respond to this, I was never alerted that there was a discussion in here / does not show as there being any active discussions in the navigation.

This is added in version 4.3.0 👍
